packages/api-explorer/src/components/JsonViewer/index.jsx

```diff
@@ -16,35 +16,37 @@ const collapseButtonStyle = {
 export default function JsonViewer({ schema, missingMessage }) {
   const [isCollapsed, setIsCollapsed] = useState(true)
 
+  if (!schema) {
```

Comment:
Schema is set as required in props, it is correct? I suppose schema should not be required

Reply:
yes. schema isn't required